File: backend/models/customer.py
from sqlalchemy import Column, Integer, String, Text
from sqlalchemy.orm import relationship
from database import Base

class Customer(Base):
    __tablename__ = "customers"
    
    id = Column(Integer, primary_key=True, index=True)
    name = Column(String(255), nullable=False)
    org_name = Column(String(255), nullable=False)
    email = Column(String(100), unique=True, nullable=False)
    phone = Column(String(20))
    address = Column(Text)

    orders = relationship("Order", back_populates="customer")

File: frontend/Models/Customer.cs
namespace frontend.Models;
using System.Text.Json.Serialization;

public class Customer
{
    public int Id { get; set; }
    public string Name { get; set; }
    [JsonPropertyName("org_name")]
    public string Organization { get; set; }
    public string Email { get; set; }
    public string Phone { get; set; }
    public string Address { get; set; }
}